UM/PluginRegistry.py

```diff
@@ -352,6 +352,10 @@ def isBundledPlugin(self, plugin_id: str) -> bool:
         self._bundled_plugin_cache[plugin_id] = is_bundled
         return is_bundled
 
+    # Indicates that a specific plugin is currently being loaded. If the plugin_id is empty, it means that no plugin
+    # is currently being loaded.
+    pluginLoadingInProgress = pyqtSignal(str, arguments = ["plugin_id"])
```

Comment:
It's a signal, so it should notify a change, not a state.

So this should be changed to pluginLoadStarted or something. If you want something that keeps track of the state, i would argue that it's a property ("pluginLoadingInProgress" makes more sense as a property)